modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/session.js

```diff
@@ -536,12 +534,20 @@ AUI.add(
 				},
 
 				_destroyBanner() {
-					var instance = this;
+					const instance = this;
+
+					const toast = document.getElementById(TOAST_ID);
+
+					const toastRootElement = toast?.parentElement;
 
-					if (Lang.isFunction(instance._banner.destroy)) {
-						instance._banner.destroy();
+					instance._banner.destroy();
+
+					if (toastRootElement) {
+						toastRootElement.remove();
 					}
 
+					Liferay.component(TOAST_ID, null);
```

Comment:
Hey, I don't understand Why is this `null` here?

Reply:
Does `Liferay.destroyComponent` can be called here?

Reply:
Yeah, that's much nicer! :+1:  Updated in https://github.com/liferay-frontend/liferay-portal/pull/1506/commits/85e70d8ba20116838fb4791dd309561bf1518847

Reply:
Nice, this ties the instance._banner lifecycle to that of the toast, which was not the case before. I was not sure if we wanted it to be that way but now it looks much better